tests/ui_tests/test_jobs_page.py

```diff
@@ -296,9 +301,14 @@ def test_job_has_correct_info_in_header_popup(
         assert (
             cluster_page.header.get_failed_job_amount_from_header() == job_info['failed_jobs']
         ), f"Failed job amount should be {job_info['failed_jobs']}"
-        assert job_info['background'] in cluster_page.header.get_jobs_circle_color(), "Bell circle should be colored"
 
-    @pytest.mark.xfail(reason="https://arenadata.atlassian.net/browse/ADCM-2048")
+        def wait_for_background():
+            assert (
+                job_info['background'] in cluster_page.header.get_jobs_circle_color()
+            ), "Bell circle should be colored"
+
+        wait_until_step_succeeds(wait_for_background, period=1, timeout=10)
```

Comment:
Maybe check more frequently? Like 0.3-0.5 secs? Or from experience, it's not likely to be less than a second?

Reply:
okay, changed)